lib/iap-validator.rb

```diff
@@ -1,34 +1,33 @@
-require "iap-validator/version"
+require 'iap-validator/version'
 
-require 'httparty'
 require 'multi_json'
+require 'typhoeus'
 
 module IAPValidator
   class IAPValidator
-    include HTTParty
+    SANDBOX_URL = 'https://sandbox.itunes.apple.com'.freeze
+    PRODUCTION_URL = 'https://buy.itunes.apple.com'.freeze
 
-    SANDBOX_URL = 'https://sandbox.itunes.apple.com'
-    PRODUCTION_URL = 'https://buy.itunes.apple.com'
+    HEADERS = {
+      'Content-Type': 'application/json'
+    }.freeze
 
-    base_uri PRODUCTION_URL
+    def self.validate(data, sandbox = false, password = nil)
+      base_uri = sandbox ? SANDBOX_URL : PRODUCTION_URL
 
-    headers 'Content-Type' => 'application/json'
-    format :json
+      json = if password.nil?
+                { 'receipt-data' => data }
+             else
+                { 'receipt-data' => data, 'password' => password }
+             end
 
-    def self.validate(data, sandbox = false, password = nil)
-      base_uri SANDBOX_URL if sandbox
+      uri = URI.join(base_uri, '/verifyReceipt')
```

Comment:
Since we're always joining this, can we just add it to the URL constants?

Reply:
Sure! Will update!

Reply:
Updated in be58755fc5759b58f1a610ab6e20d534beaa21cf